src/interfaces.ts

```diff
@@ -476,11 +476,9 @@ export interface RunnerOptions extends WorkerPoolOptions {
 export interface CronJob {
   task: string;
   payload: {
-    _cron: { ts: string; backfilled?: boolean };
```

Comment:
Why has this been removed?

Reply:
because the CronJob itself (as returned from the `make` function) doesn't contain it. In this implementation it's defined on the db layer.